### PR TITLE
:recycle: refactor: Refactored and fixed screen and components to follow app Figma design system

### PR DESCRIPTION
**Pull request by softdevandre**
## Task

This pull request refers to task LOR-XX

- [x] Does not apply to this pull request

## Description

- Refactored and fixed screen and components
- Changed app colors to follow up the pattern from design system
- Added new app launcher icons
- Created new icon button to replace google and facebook images on login screen
- Replaced LBBoyButton and LBGirlButton to LBGenderButton
- Locked screen orientation to display app vertically only
- Cleaned up useless files and code

## Types of changes #

- [ ] New feature (non-breaking change which adds functionality).
- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## What changed: #

- [x] UI
- [ ] Business rule
- [ ] Documentation
- [ ] Gradle or Build
- [ ] Navigation
- [ ] Tests
- [x] Resource files
- [ ] Project architecture
- [ ] Code style

## Checklist #

- [x] Have tested the changes.
- [x] Met all the acceptance requirements of this task.
- [x] Verified if branch is up-to-date with `development` (if not - rebase it or merge it).
- [x] Resolve git conflicts

## If any of the acceptance criteria are not met, please explain why below #
What is the criteria? Why it is different?

### CODE REVIEW

File: feature/auth/create_password/src/main/java/com/loryblu/feature/auth/create_password/navigation/CreatePasswordNavigation.kt [hunk 1]
```diff
@@ -31,9 +31,30 @@ fun NavGraphBuilder.createPasswordRoute(
...
+                onResetPasswordButtonClicked = {
+                    viewModel.verifyAllConditions("token", "")
+                },
+                shouldGoToNextScreen = shouldGoToNextScreen,
+                confirmPasswordStateValidation = { password, confirmPassword ->
+                    viewModel.confirmPasswordState(password, confirmPassword)
+                },
+                passwordStateValidation = {
+                    viewModel.passwordState(it)
+                }
```
Comment (OdisBy) [inline feature/auth/create_password/src/main/java/com/loryblu/feature/auth/create_password/navigation/CreatePasswordNavigation.kt:57]: We could simplify this later, some fix

File: feature/auth/register/src/main/java/com/loryblu/feature/auth/register/presentation/ChildRegisterScreen.kt [hunk 2]
```diff
@@ -257,41 +310,25 @@ fun ChildRegisterScreen(
...
-        genderStateValidation = {
-            GenderInput.FEMALE
-        },
-        birthdayStateValidation = {
-            BirthdayInputValid.Valid
-        },
-        apiErrorMessage = listOf(
-//            "Email inválido", "Senha deve ser maior", "Gênero não existe"
-        )
-    )
```
Comment (OdisBy) [inline feature/auth/register/src/main/java/com/loryblu/feature/auth/register/presentation/ChildRegisterScreen.kt:296]: There is no reason for remove this preview, we can add it again later

**Review by OdisBy — APPROVED** [review]
🥳 A beautiful work, congratulations for follow all the system design ❤️

File: core/ui/src/main/java/com/loryblu/core/ui/components/LBPasswordTextField.kt [hunk 3]
```diff
@@ -1,34 +1,39 @@
...
+import com.loryblu.core.ui.theme.LBSoftBlue
 import com.loryblu.core.util.validators.PasswordInputValid
 
-// TODO seek problems in this function
 /**
  * this function call the outilinedTextField for password and confirmationPassword
  * **/
 @Composable
-@OptIn(ExperimentalMaterial3Api::class)
 fun LBPasswordTextField(
```
Comment (hartmannjean) [inline core/ui/src/main/java/com/loryblu/core/ui/components/LBPasswordTextField.kt:33]: Check when user types _

**Review by hartmannjean — APPROVED** [review]
OK!

**Pull request merged**